src/renderer/services/eth2/client/interface.ts

```diff
@@ -19,8 +19,27 @@ import {
     SignedBeaconBlock,
 } from "@chainsafe/lodestar-types";
 
+export type BlockAttestations = {
```

Comment:
I don't think you need this and it's hard to maintain

Reply:
it is harder to maintain code to fit to [AttestationData](https://github.com/ChainSafe/lodestar/blob/7e067053a68bda5fede2fdde41a03637ab5edcf4/packages/lodestar-types/src/types/operations.ts#L25) as there is not ssz included for it
and after that, I will need to transform `beaconBlockRoot` from Uint8Array to a string in the saga for Attestation Effectiveness

Reply:
There is ssz for AttestationData (config.types.AttestationData)

>  I will need to transform beaconBlockRoot from Uint8Array to a string in the saga for Attestation Effectiveness

toHex(attestation.data.beaconBlockRoot)